code/datums/keybinding/mob.dm

```diff
@@ -414,3 +414,31 @@
 	if(.)
 		return
 	user.movement_locked = FALSE
+
+
+/*
+* Player input with pixel shifting currently unauthorized. Keeping for compatibility reasons
+*
+/datum/keybinding/mob/pixel_shift
+	key = "C"
+	name = "pixel_shift"
+	full_name = "Pixel shift"
+	description = "Displace your sprite within your current tile"
+	keybind_signal = COMSIG_KB_MOB_PIXELSHIFT_DOWN
+*/
+/datum/keybinding/mob/pixel_shift/down(client/user)
+	. = ..()
+	if(.)
+		return
+	var/mob/M = user.mob
+	M.shifting = TRUE
+	return TRUE
+
+/datum/keybinding/mob/pixel_shift/up(client/user)
+	. = ..()
+	if(.)
+		return
+	var/mob/M = user.mob
+	M.shifting = FALSE
+	return TRUE
```

Comment:
Remove these if they are unused and will not be used.

Reply:
removed in a3b1d4f